src/aat/java/uk/gov/hmcts/ccd/datastore/befta/DataStoreBeftaRunner.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.ccd.datastore.befta;
+
+import io.cucumber.junit.Cucumber;
+import io.cucumber.junit.CucumberOptions;
+import org.junit.BeforeClass;
+import org.junit.runner.RunWith;
+import uk.gov.hmcts.befta.BeftaMain;
+import uk.gov.hmcts.befta.BeftaTestDataLoader;
+import uk.gov.hmcts.befta.DefaultTestAutomationAdapter;
+
+
+@RunWith(Cucumber.class)
+@CucumberOptions(plugin = "json:target/cucumber.json",
+    glue = "uk.gov.hmcts.befta.player",
+    features = {"classpath:features"})
+public class DataStoreBeftaRunner extends BeftaMain {
+
+    @BeforeClass
+    public static void init() {
+        setTaAdapter(new DefaultTestAutomationAdapter());
+        BeftaTestDataLoader.main(null);
```

Comment:
Apart from the data loading, there is also another tear-up logic to inject syntactic flexibilities into DSL elements. That is inside BeftaMain. We should probably combine any such initial tear up logic into a dedicated public static method inside befta main and re-use that from anaywhere needed. The adapted object can be a parameter to that one. This way, we can standardise and re-use the logic to max extent.

Reply:
so we need to change ccd-api-test-automation-fw for this change then

Reply:
yep..

Reply:
Changed in ccd-api-test-automation-fw  and calling the necessary methods